packages/manager/apps/telecom/src/app/telecom/pack/move/eligibility/lineNumber/move-eligibility-lineNumber.html

```diff
@@ -33,12 +33,154 @@
             </span>
         </span>
     </div>
-    <button
-        type="submit"
-        class="btn btn-primary"
-        data-translate="submit"
-        data-ng-disabled="$ctrl.loading || availabilityForm.phoneNumber.$invalid || $ctrl.method !== 'number'"
-    ></button>
-    <oui-spinner class="ml-2" data-ng-if="$ctrl.loading" data-size="s">
-    </oui-spinner>
+    <div class="mt-3">
+        <button
+            type="submit"
+            class="btn btn-primary"
+            data-translate="submit"
+            data-ng-disabled="$ctrl.loading || availabilityForm.phoneNumber.$invalid || $ctrl.method !== 'number'"
+        ></button>
+        <oui-spinner class="ml-2" data-ng-if="$ctrl.loading" data-size="s">
+        </oui-spinner>
+    </div>
+
+    <div data-ng-if="$ctrl.displayFirstStep" class="mt-3">
+        <div class="row">
+            <div class="col-md-6" data-ng-repeat="copper in $ctrl.copperOffers">
+                <div class="oui-box oui-box_light">
+                    <h4
+                        class="oui-box__heading"
+                        data-ng-if="copper.status === $ctrl.STATUS.active"
+                        data-translate="pack_move_eligibility_line_status_active"
+                    ></h4>
+                    <h4
+                        class="oui-box__heading"
+                        data-ng-if="copper.status === $ctrl.STATUS.inactive"
+                        data-translate="pack_move_eligibility_line_status_inactive"
+                    ></h4>
+
+                    <div data-ng-if="copper.errorMessage">
```

Comment:
No `oui-message` for this ?

Reply:
No it's a message to inform user that there is an error on eligibility test on the line number like "No active line found".